tslint.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": "@oclif/tslint"
+  "extends": "@heroku-cli/tslint"
```

Comment:
OCLIF/core includes ESLint, so we shouldn't need TSLint at all, and this file can be deleted. Reference: https://oclif.io/blog/2019/12/05/oclif-eslint-migration

Reply:
I'm not sure that we want to do this. oclif/core's eslint settings are not something I care for. I suspect we shouldn't be using them in `@heroku-cli/command` either.